projects/test_project/rules.mk

```diff
@@ -0,0 +1,8 @@
+# Specify the device family to build for
+DEVICE_FAMILY := stm32f0xx
+
+# Specify the main file (relative to this file if not in this folder)
+MAIN := main.c
+
+# Specify the device library you want to include
+DEVICE_LIBRARY := libstm32f0.a
```

Comment:
I would make this a list of libraries (without extension) so we don't need to special-case our libraries.